Sources/Log.swift

```diff
@@ -0,0 +1,102 @@
+import Foundation
```

Comment:
This is from Alicerce, which has is own logging layer. this is what the dependency `swift-log` does. You don't need this